helmfile.d/0100.external-dns.yaml

```diff
@@ -27,7 +27,7 @@ releases:
   wait: true
   values:
     - extraEnv:
-        EXTERNAL_DNS_SOURCE: "service\ningress"
+      EXTERNAL_DNS_SOURCE: "service\ningress"
```

Comment:
So, looks like the format has changed to this: 

https://github.com/helm/charts/blob/master/stable/external-dns/values.yaml#L93-L95